File: Backend/BetaCycle/Models/Order.cs
using System;
using System.Collections.Generic;

namespace BetaCycle.Models;

public partial class Order
{
    public long TransactionId { get; set; }

    public long OrderId { get; set; }

    public long UserId { get; set; }

    public long ProductId { get; set; }

    public int Quantity { get; set; }

    public double ProductPrice { get; set; }

    public long IdPayment { get; set; }

    public string? Status { get; set; }

    public long AddressId { get; set; }

    public virtual Product Product { get; set; } = null!;

    public virtual Transaction Transaction { get; set; } = null!;
}

File: BetacycleFrontend/src/app/shared/models/order.ts
import { Product } from "./product"

export class Order{
        userId:number = 0
        productId:number = 0
        quantity:number = 0
        productPrice:number = 0
        idPayment:number = 0
        status: string = "Storage"
        transactionId:number = 0
        orderId:number = 0
        addressId: number = 0
        transaction = {}
        product:Product = new Product()
        date:string = '2024-01-01'
}